### PR TITLE
New ways to make client

### PR DESCRIPTION
**Pull request by fear-the-reaper**
# Description
    Added `NewIntelOwlClientThroughJsonFile` to make the `IntelOwlClient` through a JSON file. Furthermore, changed the type of `Timeout` field in `IntelOwlClientOptions` from `time.Duration` to `uint64` as `time.Duration` can't be marshalled and unmarshalled.

## Issues
add the related issue(s) your change tackles

## Type of change
list down the type of change
- [ ] Bug fix
- [ ] New feature (added a new endpoint or created)
- [ ] Breaking fix (a bug fix or new feature that would cause exisiting functionality not to work as expected)

## Checklist
- [ ] if a new feature was added it passed all its (if made) test cases 
- [ ] if a bug fix was added it passed all its existing test cases 

### Important Rules
- If your changes decrease the overall tests coverage (you will know after the Codecov CI job is done), you should add the required tests to fix the problem
- Everytime you make changes to the PR and you think the work is done, you should explicitly ask for a review

Please delete if the PR is for bug fixing.

### CODE REVIEW

File: gointelowl/client.go [hunk 1]
```diff
@@ -37,11 +38,11 @@ type successResponse struct {
...
 type IntelOwlClientOptions struct {
-	Url   string
-	Token string
+	Url   string `json:"url"`
+	Token string `json:"token"`
 	// * so basically your SSL cert: path to the cert file!
-	Certificate string
-	Timeout     time.Duration
+	Certificate string `json:"certificate"`
+	Timeout     uint64 `json:"timeout"`
```
Comment (sp35) [inline gointelowl/client.go]: Is this timeout in seconds or milli-seconds? Let's document this.

**Review by sp35 — APPROVED** [review]
LGTM, you can merge the PR if it's ready.

File: gointelowl/client.go (hunk 1 above)
Comment (fear-the-reaper) [reply gointelowl/client.go]: Seconds. Yeah I'll make a comment that it is in seconds.

**Review by sp35 — APPROVED** [review]
LGTM

**Pull request merged**